.github/workflows/build_and_test.yaml

```diff
@@ -0,0 +1,66 @@
+# Please reference work here https://github.com/EVerest/everest-core/tree/main/.github/workflows
+# TODO: modify to reuse the above workflow to DRY up CI.
+
+name: Build and test libtimer
+on: 
+  pull_request: {}
```

Comment:
```suggestion
on: 
  pull_request: {}
  push:
```
```suggestion
on: 
  pull_request: {}
  push:
```

Reply:
@couryrr-afs, could you please add this temporary to "register" the workflow to github, so that we can run it in this pull request?